deploy/crds/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -80,9 +84,23 @@ spec:
                           cron authpriv ftp ntp security console solaris-cron     local0
                           local1 local2 local3 local4 local5 local6 local7"
                         type: string
+                      msgId:
```

Comment:
Capitalization: msgID - see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md . It's not 100% clear to me if ID is an acronym or an abbreviation, but I grepped the k8s.io repos and they always use "ID"